cmake/FindFCGI.cmake

```diff
@@ -0,0 +1,72 @@
+# Created by Viacheslav Greshilov http://github.com/greshilov
+
+set(
+  _FCGI_SOURCE_PATHS
+  /usr/
+  /usr/local/
+  /usr/local/Cellar/fcgi/2.4.0/
```

Comment:
Is it possible to avoid a hard-coded version here? Maybe some wildcards with a greater version selection?

Reply:
Fixed :)